app/uk/gov/hmrc/tariffclassificationfrontend/models/AppealStatus.scala

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.tariffclassificationfrontend.models
+
+object AppealStatus extends Enumeration {
+  type AppealStatus = Value
+  val IN_PROGRESS, ALLOWED, DISMISSED = Value
+
+  def format(status: Option[AppealStatus]): String = {
+    status match {
+      case Some(IN_PROGRESS) => "Under appeal"
+      case Some(ALLOWED) => "Appeal allowed"
+      case Some(DISMISSED) => "Appeal dismissed"
+      case None => "None"
+      case _ => ???
```

Comment:
Will this blow-up if we ever have a new appeal status?  Is this the equivalent of "unexpected value exception"?

Reply:
Thought so.  Nice.